packages/hydrogen-codegen/src/pluck.ts

```diff
@@ -11,7 +11,7 @@ export const pluckConfig = {
     // Check for internal gql comment: const QUERY = `#graphql ...`
     const hasInternalGqlComment =
       node.type === 'TemplateLiteral' &&
-      /\s*#graphql\s*\n/i.test(node.quasis[0]?.value?.raw || '');
+      /\s*#graphql:?\w*\s*\n/i.test(node.quasis[0]?.value?.raw || '');
```

Comment:
@frandiox making sure this change is ok to make now assuming the multiple schema in one file PR wont make it in?

Reply:
So I don't think we should make this change yet. We left the `:` out of this regexp on purpose because we basically use the `:customer` part to **disable codegen** in secondary examples (because we know `:` is not supported yet, and are expecting it will start working once this syntax is official): https://github.com/Shopify/hydrogen/blob/d6d01322b430761c6ac3be71aa8fee798c85de37/examples/customer-api/app/routes/_index.tsx#L8

Plus, until the syntax becomes official, I think we should not include `#graphql:customer` in our starter template since it might be confusing -- as mentioned, we only use this in secondary examples just to actually disable codegen 😅 

Sooo, I'd say let's wait until https://github.com/graphql/graphiql/pull/3411 is merged to legitimate our use case.